catalyst_voices/packages/internal/catalyst_voices_models/lib/src/markdown_string.dart

```diff
@@ -0,0 +1 @@
+extension type const MarkdownString(String data) implements Object {}
```

Comment:
How about calling it just Markdown? Or Markdown data?

Reply:
Lets use `MarkdownData`. `Markdown` is format and widget which may be confusing